src/test_l1_deepSleepMgr.c

```diff
@@ -175,15 +167,15 @@ void test_l1_deepSleepMgr_negative_PLAT_DS_INIT (void)
 
     // Variation 01: Call PLAT_DS_INIT() and check the result
     result = PLAT_DS_INIT();
-    UT_ASSERT_EQUAL(result, DEEPSLEEPMGR_SUCCESS);
+    UT_ASSERT_EQUAL_FATAL(result, DEEPSLEEPMGR_SUCCESS);
 
     // Variation 02: Call PLAT_DS_INIT() again and expect it to be already initialized
-     result = PLAT_DS_INIT();	
-     CHECK_FOR_EXTENDED_ERROR_CODE( result, DEEPSLEEPMGR_ALREADY_INITIALIZED, DEEPSLEEPMGR_SUCCESS);
+    result = PLAT_DS_INIT();	
+    CHECK_FOR_EXTENDED_ERROR_CODE( result, DEEPSLEEPMGR_ALREADY_INITIALIZED, DEEPSLEEPMGR_SUCCESS);
```

Comment:
In Extended error code we are not using UT_ASSERT_EQUAL_FATAL

so in init() api how can we handle it ?

Reply:
Currently UT_ASSERT_EQUAL_FATAL is not handled for extended error codes.

Reply:
I think most of the extended error is used for negative cases and API is expected to fail. No need of using FATAL here

Reply:
Ok